### DISCUSSION TITLE
A valid external Instance reference no longer exists.

### DISCUSSION THREAD

**Discussion by Matze99** [gpuweb/gpuweb#4764] [Q&A] [answered]
Whenever I execute anything that uses WebGPU, it fails and it usually only returns the warning `A valid external Instance reference no longer exists.` This is triggered when the GPU device generated by using `const device = await adapter.requestDevice();` is lost. I was able to resolve the issue on Windows by disabling Vulkan in the flags. The problem is that I need to run this on a linux device on which the vulcan disabling does not resolve the issue. When the adapter is lost, the reason is listed as `unknown`.

<img width="304" alt="image" src="https://github.com/user-attachments/assets/f6c9cd7a-517a-4b84-9bbe-eb6012867760">

I searched for similar issues in the discussions section and on the web in general. The only thing I could find is [this](https://github.com/gpuweb/gpuweb/discussions/4676). This discussion is related to memory issues, which would be reflected in the lost device reason. Since this is unknown in my case, it is unlikely that the reason is related to the other discussion.

**Comment by greggman** [comment, marked as answer]
I'm just going repeat what was written in the question you linked to

This is a Chromium-specific message and it is not clear without more information might be causing it. If you believe that it is a browser bug, please file an issue on [issues.chromium.org](https://issues.chromium.org) with detailed repro steps.
